packages/web-components/fast-components/src/combobox/combobox.styles.ts

```diff
@@ -15,10 +15,6 @@ export const ComboboxStyles = css`
         user-select: none;
     }
 
-    :host(:active) .selected-value {
-        user-select: none;
```

Comment:
I wonder if it makes more sense setting "functional" styles like this should be done in foundation (ie. in component or template) vs. in the styled component packages.

Reply:
The block here still needs to be removed, but that would probably make sense for the block above on line 15.